esmvalcore/_recipe.py

```diff
@@ -332,11 +332,19 @@ def _get_default_settings(variable, config_user, derive=False):
 
     # Configure CMOR metadata check
     if variable.get('cmor_table'):
+        raise_exception = True
+        report_only_warning = False
+        if config_user['cmor_checks_type'] in ['none', 'relaxed']:
+            raise_exception = False
+            if config_user['cmor_checks_type'] == 'relaxed':
+                report_only_warning = True
         settings['cmor_check_metadata'] = {
             'cmor_table': variable['cmor_table'],
             'mip': variable['mip'],
             'short_name': variable['short_name'],
             'frequency': variable['frequency'],
+            'raise_exception': raise_exception,
```

Comment:
Two extra arguments seems overkill, just one argument:
`'raise_on': config_user['cmor_checks'],`
should be sufficient.

Reply:
Or something similar, maybe the levels could be translated to `'never', 'warning', 'error'` or something similar here..

Reply:
no, the reason behind this is to recycle the functionality for dry-run where we don't necessarily need to use the `--cmor-checks` flag, I should have explained it better

Reply:
Please do?